deployments/types/common.json

```diff
@@ -130,5 +130,9 @@
 	"messages_in_oldest_entry": "MessageNonce",
 	"total_messages": "MessageNonce"
   },
-  "AncestryProof": "()"
+  "AncestryProof": "()",
+  "MessageFeeData": {
+	"lane_id": "LaneId",
```

Comment:
not a blocker at all but it would be nice to have it indented

Reply:
Interesting :D I've re-formatted the entire file, cause it wasn't using the right indentation settings from `.editorconfig`. Should be better now.